test/e2e/network/ingress.go

```diff
@@ -1141,6 +1142,14 @@ var _ = SIGDescribe("Ingress API", func() {
 		framework.ExpectNoError(err)
 		framework.ExpectEqual(updatedStatus.Status.LoadBalancer, statusToUpdate.Status.LoadBalancer, fmt.Sprintf("updated object expected to have updated loadbalancer status %#v, got %#v", statusToUpdate.Status.LoadBalancer, updatedStatus.Status.LoadBalancer))
 
+		ginkgo.By("get /status")
+		ingResource := schema.GroupVersionResource{Group: "networking.k8s.io", Version: ingVersion, Resource: "ingresses"}
+		gottenStatus, err := f.DynamicClient.Resource(ingResource).Namespace(ns).Get(context.TODO(), createdIngress.Name, metav1.GetOptions{}, "status")
+		framework.ExpectNoError(err)
+		statusUID, _, err := unstructured.NestedFieldCopy(gottenStatus.Object, "metadata", "uid")
+		framework.ExpectNoError(err)
+		framework.ExpectEqual(string(createdIngress.UID), statusUID, fmt.Sprintf("createdIngress.UID: %v expected to match statusUID: %v ", createdIngress.UID, statusUID))
```

Comment:
Is the string formatting needed here over something like [this](https://github.com/kubernetes/kubernetes/blob/31962804c0e0fc12319437febd46c9ce45f4ef0e/test/e2e/network/ingress.go#L1066)

Reply:
I tried the other option first but found that `createdIngress.UID` was `types.UID` when trying to match it with `statusUID` which was `string`